### ISSUE TITLE
Server crashes with "Request textDocument/formatting failed."

### ISSUE THREAD

**Issue by Glaisher** [johnsoncodehk/volar#2077]
**Environment**
WSL Ubuntu
Volar v1.0.9
VS Code v1.72.2

Whenever a vue file is saved, I am getting the following error.

Request textDocument/formatting failed.
  Message: Request textDocument/formatting failed with message: Overlapping edit
  Code: -32603

It does work when the vue language server is restarted, but only one edit goes through successfully after restart each time.

**Comment by Glaisher** [comment]
```
[2022-10-30 19:35:32.839] [exthost] [error] [Vue.volar] provider FAILED
[2022-10-30 19:35:32.839] [exthost] [error] Error: Request textDocument/formatting failed with message: Overlapping edit
	at Ys (/home/acc/.vscode-server/extensions/vue.volar-1.0.9-linux-x64/dist/node/client.js:4:228)
	at Fn (/home/acc/.vscode-server/extensions/vue.volar-1.0.9-linux-x64/dist/node/client.js:3:6865)
	at Immediate.<anonymous> (/home/acc/.vscode-server/extensions/vue.volar-1.0.9-linux-x64/dist/node/client.js:3:6712)
	at processImmediate (node:internal/timers:466:21)
```

**Comment by tazyong** [comment]
I encountered the same problem. An error occurred when Volar v1.0.9 was formatting the code. It can only be normal if it is downgraded to v1.0.8.

![image](https://user-images.githubusercontent.com/24738011/199471790-3c79a170-0757-4983-b9e6-7503f29a09fb.png)

**Comment by roblues** [comment]
same.
when the problem is occurred. reopen vue file, it is work.

**Comment by xinde** [comment]
same

**Comment by baixjian** [comment]
same

**Comment by helderjnpinto** [comment]
same issue

**Comment by nxzz** [comment]
same

**Comment by helderjnpinto** [comment]
same

**Comment by wycats** [comment]
This happened to me when I **added** a `<style scoped>` tag to an existing `.vue` file. When I reopened the file, the problem went away.

**Comment by johnsoncodehk** [comment]
Can you reproduce this problem in https://github.com/johnsoncodehk/volar-starter?

**Comment by cesswhite** [comment]
I have the same problem, and closing and opening the same file does not work for me.

<img width="543" alt="Captura de Pantalla 2022-11-29 a la(s) 09 06 35" src="https://user-images.githubusercontent.com/44076866/204569272-101a4537-b69a-42e4-bb27-941af61492d5.png">

<img width="702" alt="Captura de Pantalla 2022-11-29 a la(s) 09 07 05" src="https://user-images.githubusercontent.com/44076866/204569324-6ece584c-54dd-47d9-9570-5fc453795100.png">

**Comment by johnsoncodehk** [comment]
Please provide minimal reproduction code so that I can debug the problem.

**Comment by whidy** [comment]
> Please provide minimal reproduction code so that I can debug the problem.

I found this error and try to reproduce it.

clone the project: https://github.com/whidy/issues-with-vue3/tree/volar-formatting-failed-demo

maybe video would help: https://www.youtube.com/watch?v=BB92-GNgmmQ

plugins in the video all lastest update.

**Comment by tjventurini** [comment]
Same here. It is driving me nuts 😭

@johnsoncodehk Not sure what causes the problem. So I can't tell you how to reproduce it. Is there anything out of the blue that you would look at and that I can provide? 🤞

**Comment by mileusna** [comment]
I had the same issue in some files. After I have removed some commented code it worked. It looks like something is confusing extension when there are some empty lines **with tabs** between comments.

I can't actually tell what it is, but when I removed this comment, or just removed empty lines between commented lines, **or just tabs on this empty lines**, it worked.

```
// computed(function () {
    
//     if (query.value === '') {
//         return values;
//     }

//     return values;

// })
```

I hope this will help.
